javaagent/Dockerfile

```diff
@@ -3,4 +3,4 @@ FROM eclipse-temurin:11-jre
 ADD build/libs/app.jar /app.jar
 ADD build/agent/opentelemetry-javaagent.jar /opentelemetry-javaagent.jar
 
-ENTRYPOINT java -jar -javaagent:/opentelemetry-javaagent.jar /app.jar
+ENTRYPOINT java -javaagent:/opentelemetry-javaagent.jar -jar /app.jar
```

Comment:
the `-jar -javaagent` ordering was causing a manifest error trying to load the jar